frontend/src/metabase/parameters/components/widgets/DateRangeWidget.jsx

```diff
@@ -1,38 +1,44 @@
 import React, { Component } from "react";
+import PropTypes from "prop-types";
```

Comment:
Would it make sense to convert this stuff to flow here instead of continuing to use PropTypes?